### ISSUE TITLE
Cannot expand home directory to choose folder for either media or playlists.ary 

### ISSUE THREAD

**Issue by w06lmy** [LMS-Community/slimserver#869]
During initial setup and and also when I visit settings after completing the initial setup, I am unable to expand the home folder in order select subfolders in within the /home directory.   I am able to expand other folders that contain subfolders.  I am running linux, so in the home directory are subfolders for users and, for each user, at least a Music folder.  I want to use /home/username/Music, but cannot access it in the server settings.  If I click the plus sign to expand, it disappears, and home is selected.  Not acceptable.  If I click on other folders in the root directory that contain subfolders, I am able to open all the way down to the lowest level that doesn't contain a subfolder.

I am running kubuntu 2022.04.1.  I had problems installing it, as discussed in another thread, so I assumed a faulty installation.  I installed Fedora 37, which installed with no problem, but I also cannot access subfolders of the home directory with Fedora.

I am trying to install in my own /home/user/Music directory.

**Comment by michaelherger** [comment]
Most likely you're not looking at your home, but the `slimserver` user's - which might be empty. LMS is not running as your user.

**Comment by w06lmy** [comment]
/home is the folder containing all users' home pages.  No subfolders, such as /home/user1 or /home/user2 are visible when I click on the plus sign for the listing "home" that appears in the dialog box for selecting a media folder.

I have confirmed that the user home directory /home/user/Music does appear.  I confirmed that on my previous installation of LMS, on kubuntu 2018.04, LMS is using the music folder in my home directory, /home/user/Music.   

I am attaching two images.  Screenshot 1 shows the dialog box for selecting a folder before I select home, and screenshot 2 shows the dialog box after I select home.  

![Screenshot_of_folder_choicesf_2](https://user-images.githubusercontent.com/65308112/224450863-078fbeb0-1a0c-4306-b47a-39620e9c8c74.png)

![Screenshot_of_folder_choicesf_1](https://user-images.githubusercontent.com/65308112/224450834-ecd4b7ca-5746-40e9-9f0f-047a4da72a4a.png)
